libft/sources/memory/ft_free.c

```diff
@@ -6,37 +6,17 @@
 /*   By: inikulin <inikulin@student.42berlin.de>    +#+  +:+       +#+        */
 /*                                                +#+#+#+#+#+   +#+           */
 /*   Created: 2024/09/04 18:43:58 by inikulin          #+#    #+#             */
-/*   Updated: 2024/10/06 22:40:21 by inikulin         ###   ########.fr       */
+/*   Updated: 2024/10/23 20:10:09 by inikulin         ###   ########.fr       */
 /*                                                                            */
 /* ************************************************************************** */
 
 #include "../../libft.h"
 
-void	ft_free_s_null(void **c)
-{
-	ft_free_s(*((char **)c));
-	*c = 0;
```

Comment:
If it is nupp pointer i'd prefer to assign to NULL. its how we understand that its null pointer

Reply:
If c is a nullptr, this line will cause a segfault. That's why I added the check in the new version in another file.